pkg/cmd/flags/logger.go

```diff
@@ -11,35 +11,56 @@ import (
 	"github.com/aquasecurity/tracee/pkg/logger"
 )
 
+const (
+	LogFlagShort                = "l"
+	LogFlag                     = "log"
+	LogLevel                    = "level"
+	LogLevelDebug               = "debug"
+	LogLevelInfo                = "info"
+	LogLevelWarn                = "warn"
+	LogLevelError               = "error"
+	LogLevelFatal               = "fatal"
+	LogFile                     = "file"
+	LogFilter                   = "filter"
+	LogInclude                  = "include"
```

Comment:
```suggestion
	LogFilterInclude                  = "include"
```

The same pattern for following.